docs/admin/access.rst

```diff
@@ -402,30 +402,30 @@ VCS
 
     Update the internal repository [`Administration`, `Manage repository`]
 
-Global privileges
-    Use management interface (global)
+Site wide privileges
+    Use management interface
 
-    Add language definitions (global)
+    Add language definitions
```

Comment:
These should match in app permission names, but it seems to be already out of sync now since https://github.com/WeblateOrg/weblate/pull/2895:

https://github.com/WeblateOrg/weblate/blob/69aa093dfd7c74e94fe2b04e181c31bd54e0bdbb/weblate/auth/data.py#L123-L142

Reply:
Changed those. I like entire, but in English /project wide/site wide is neat.